cerbos.yaml

```
@@ -33,21 +33,33 @@ pipeline:
       output: cerbos
       packages: ./cmd/cerbos
 
-  - uses: go/build
-    with:
-      ldflags: |-
-        -X github.com/cerbos/cerbos/internal/util.Version=${{package.version}}
-        -X github.com/cerbos/cerbos/internal/util.Commit=$(git rev-parse HEAD)
-        -X github.com/cerbos/cerbos/internal/util.BuildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
-      output: cerbosctl
-      packages: ./cmd/cerbosctl
-
 subpackages:
+  - name: ${{package.name}}ctl
```

Let's change it to `istioctl` for more clarity. Non-blocker but based on other examples.
```suggestion
  - name: cerbosctl
```